pkg/bff/config/config.go

```diff
@@ -33,6 +33,8 @@ type Config struct {
 	LogLevel     logger.LevelDecoder `split_words:"true" default:"info"`
 	ConsoleLog   bool                `split_words:"true" default:"false"`
 	AllowOrigins []string            `split_words:"true" default:"http://localhost,http://localhost:3000,http://localhost:3003"`
+	RegisterURL  string              `split_words:"true" required:"true"`
+	LoginURL     string              `split_words:"true" required:"true"`
```

Comment:
These configuration values will have to be added when we do the next staging deployment.

Reply:
Should we add a little inline comment reminder that trailing slashes are not allowed for these two values?